src/rest/app.py

```diff
@@ -0,0 +1,178 @@
+import json
+import logging
+import time
+import uuid
+from datetime import datetime
+
+from fastapi import (Depends, FastAPI, File, Form, HTTPException, Security,
+                     UploadFile)
+from fastapi.responses import JSONResponse, PlainTextResponse
+from fastapi.security.api_key import APIKeyHeader
+from pydantic import BaseModel
+from pydub import AudioSegment
+from starlette.status import HTTP_401_UNAUTHORIZED
+
+from src.helper.config import CONFIG
+from src.helper.data_handler import DataHandler
+from src.helper.types.transcription_status import TranscriptionStatus
+
+LOGGER = logging.getLogger(__name__)
+DATA_HANDLER = DataHandler()
+
+
+class TranscriptionData(BaseModel):
+    transcription_id: str
+    status: str
+    start_time: str
+    settings: dict | None = None
+    model: str | None = None
+    task: str
+    text: str | None = None
+    language: str | None = None
+
+
+config = CONFIG
+
+app = FastAPI()
+api_key_header = APIKeyHeader(name="Authorization", auto_error=False)
+
+
+def custom_openapi():
```

Comment:
This is missing the "what does it do?" comment. 
I don't think they are necessary per se but present everywhere else...therefore it is great for unified code style